Gemfile

```diff
@@ -0,0 +1,31 @@
+source "https://rubygems.org"
+
+gem "bcrypt", "~> 3.1.7"
+gem "bootsnap", require: false
+gem "cancancan"
```

Comment:
Let's remove this gem from here. We will add the gem in the PR where it is implemented